west.yml

```diff
@@ -42,7 +42,8 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 31529f68ec7fb266a4ec1761009e100ef9850ea5
+      revision: 1847721da924e6819df30248105d83a88a15285c
+      remote: thst
```

Comment:
and if you do as described in https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/813/files/582cf045492b64f21a75cf8fc564e83b016bd86b..8a083f232838b80c6a6c7a40a20897020c1dfc64#r297865946
then you don't need to specify your own remote, but can default to `ncs` :)

Thus this line can be removed.